docs/explainer.md

```diff
@@ -0,0 +1,88 @@
+# MiniApp Widget Requirement explainer
+
+> Note: This document serves as a supplementary explanation of the MiniApp Widget Requirement note. If there is any inconsistency with the note, you should consider the note to be authoritative.
+
+## Authors
+Chen Canfeng (Xiaomi)  
+Zhou Bingqing (Xiaomi)  
+Chen Yinli (Xiaomi)
+
+## What is this?
+The MiniApp Widget is a special form of MiniApp Page. Same as a page, a widget runs in a host environment that is called the User Agent. Unlike a page, a widget can occupy a certain area instead of the entire screen. The Widget Requirement document describes the initial design considerations for developing a MiniApp Widget. The detailed Widget specification will be described in another document called Widget Spec.
```

Comment:
```suggestion
The MiniApp Widget is a special form of MiniApp Page. Same as a page, a widget runs in a host environment that is called the User Agent. Unlike a page, a widget can occupy a certain area instead of the entire screen. The Widget Requirements document describes the initial design considerations for developing a MiniApp Widget. The detailed Widget specification will be described in another document called Widget Spec.
```